app/src/main/java/com/example/mylittlelibrary/RetrofitClientInstance.kt

```diff
@@ -12,7 +12,7 @@ object RetrofitClientInstance {
         get() {
             if (retrofit == null) {
                 val gson = GsonBuilder().setDateFormat("yyyy-MM-dd'T'HH:mm:ss").create()
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
Code removed that Android Studio highlighted and says is optional and redundant, which is what kotlinlang.org says to do with optional and redundant code.
https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs